charts/newrelic-infrastructure/templates/clusterrole.yaml

```diff
@@ -18,6 +18,7 @@ rules:
       - "services"
       - "nodes"
       - "namespaces"
+      - "pods"
```

Comment:
is this in place to fix existing issues with rbac? if so, this should just be a separate PR that we can merge without the fix being blocked by the windows work

Reply:
from discussion: this seems to be specific to windows since it's still using infra agent v2. so let's add a conditional or at least code comment

Reply:
Adding more details here, 
V2 integration has “pod” in rbac resources. But its removed in V3. Since we are using windows integration same as in V2,  I have added “pod” in rbac.  With out this change, we get follwoing error from Windows agent
time="2023-05-12T10:30:40Z" level=error msg="Integration command failed" error="exit status 1" instance=nri-kubernetes integration=com.newrelic.kubernetes prefix=integration/com.newrelic.kubernetes stderr="time=\"2023-05-12T10:30:40Z\" level=panic msg=\"failed to discover nodeIP with kube-state-metrics, got error: pods is forbidden: User \\\"system:serviceaccount:newrelic:newrelic-newrelic-infrastructure\\\" cannot list resource \\\"pods\\\" in API group \\\"\\\" at the cluster scope\"\ntime=\"2023-05-12T10:30:40Z\" level=fatal msg=\"failed to discover nodeIP with kube-state-metrics, got error: pods is forbidden: User \\\"system:serviceaccount:newrelic:newrelic-newrelic-infrastructure\\\" cannot list resource \\\"pods\\\" in API group \\\"\\\" at the cluster scope\"\n" working-dir="C:\\Program Files\\New Relic\\newrelic-infra\\newrelic-integrations”